azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+pr:
+- main
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+
+resources:
+  repositories:
+  - repository: buildimage
+    type: github
+    name: sonic-net/sonic-buildimage
```

Comment:
Please change the build repository to sonic-net/sonic-platform-vpp.

As part of the sonic-platform-vpp build sonic-buildimage is pulled.

Reply:
This is using the same build options with public SONiC image building.
It is used in Line:19, Line:20
  - template: /.azure-pipelines/template-variables.yml@buildimage
  - template: /.azure-pipelines/azure-pipelines-repd-build-variables.yml@buildimage